docs/roles/OBORoles.md

```diff
@@ -0,0 +1,23 @@
+# OBO Foundry Roles
+
+OBO Foundry Roles are designed to manage a single aspect of the day-to-day activities of the OBO Foundry. 
+This document presents the SOPs for role recruitment and retirement, and the expectations and responsibilities common to all roles.
+
+## General SOPs for role recruitment and retirement
+
+- The duration of an appointment to a role is 1 year, with the possibility of extension.
+- Role assignments will be reviewed during the first OBO Operations Committee call that falls in December each year.
+- Any OBO Operations Committee member may nominate a candidate for a given role. Alternatively, candidates can be self-nominated. New volunteers are particularly encouraged to participate.
+- Candidates for each role should read this document and all documents pertaining to the role of interest, acknowledge that they understand the involved duties, and agree to fulfill them to the best of their ability.
+- Each candidate must submit an application for a role to the OBO Operations Committee. The application should outline the motivation for taking on the role as well as 
+  describe the skillsets that will make them suitable for that role.
+- Applications will be reviewed by the OBO Operations Committee. Decisions for appointing a member of the community to a role will be based on vote. A positive outcome occurs (that is, the appointment is approved) if the candidate receives at least 70% in favor.
+- Any OBO Operations Committee member may suggest that an individual be removed from a role. In such case, a vote must be called to enact the removal. A postive outcomes occurs (that is, the removal is implemented) if at least 70% of the OBO Operations Committee members are in favor of removal.
```

Comment:
here you explicitly say "70% of the obo ops committee members", suggesting that you'd need 14 of 20 to vote yes, even if only 10 came to the meeting. in that case, how would the remaining votes be gathered? will votes be anonymous? what if a random non-committee member is on the call, and the vote is about someone they know?

Reply:
```suggestion
- Any OBO Operations Committee member may suggest that an individual be removed from a role. In such case, a vote must be called to enact the removal _during an OBO Operations Call_. A postive outcomes occurs (that is, the removal is implemented) if at least 70% of the OBO Operations Committee members _present in the call_ are in favor of removal.
```

Reply:
What do you think? This excluded non-OBO ops people that happen to be on a call.